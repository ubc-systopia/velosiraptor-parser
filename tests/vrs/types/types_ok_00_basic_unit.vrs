extern type Bar {
    field: int
}

osspec Foo {
    extern type Bar {

    }

    extern type Bar {
        field: int
    }

    extern type Bar {
        field: int,
        field: bool,
    }


    extern type Bar {
        field: int,
        field: Baz,
    }
}