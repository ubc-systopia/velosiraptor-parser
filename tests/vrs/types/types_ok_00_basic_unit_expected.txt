VelosiParseTree(tests/vrs/types/types_ok_00_basic_unit.vrs)
---------------------------------------------
extern type Bar {
  field: int
}


osspec Foo {
  extern type Bar {
  }

  extern type Bar {
    field: int
  }

  extern type Bar {
    field: int
    field: bool
  }

  extern type Bar {
    field: int
    field: Baz
  }

}


---------------------------------------------
